cache-config/testing/docker/docker-compose.yml

```diff
@@ -57,6 +57,9 @@ services:
     ports: 
       - "443:443"
     build:
+      args:
+        - OS_DISTRO==${OS_DISTRO:-rockylinux/rockylinux}
```

Comment:
Rocky Linux now has an official image, `rockylinux/rockylinux` can be just `rockylinux`.